mne_bids/mne_bids.py

```diff
@@ -697,8 +697,8 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     _participants_tsv(raw, subject_id, participants_tsv_fname, overwrite,
                       verbose)
     _participants_json(participants_json_fname, True, verbose)
-    _scans_tsv(raw, os.path.join(kind, bids_fname), scans_fname,
-               overwrite, verbose)
+    _scans_tsv(raw, '/'.join([kind, bids_fname.replace(os.sep, '/')]),
+               scans_fname, overwrite, verbose)
```

Comment:
still a bit complicated ;-) Would it be simpler if it's done inside _scans_tsv? If not, I can live with this

Reply:
yeah true. I think initially I was thinking it would be needed in other places, but that doesn't seem to be the case...